staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -599,7 +599,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	if a.group.MetaGroupVersion != nil {
 		reqScope.MetaGroupVersion = *a.group.MetaGroupVersion
 	}
-	if a.group.OpenAPIModels != nil && utilfeature.DefaultFeatureGate.Enabled(features.ServerSideApply) {
+	if a.group.OpenAPIModels != nil {
```

Comment:
Not necessarily in this pull-request, but I think we need to remove that condition. the `scope.FieldManager` should always be set otherwise the handler is going to crash later when it's used, which is a worse experience. 

If this causes flakiness due to extra memory-allocation, @alexzielenski fix for swagger unmarshaling will almost certainly solve the problem.

cc @Jefftree